nominations/views.py

```diff
@@ -233,17 +237,34 @@ def form_valid(self, form):
         application.questionnaire.save()
 
         plaintext = get_template('email/candidate_email.txt')
-        htmly     = get_template('email/candidate_email.html')
+        htmly = get_template('email/candidate_email.html')
 
-        d = Context({'group': group,'candidate_name':candidate_name,'group_rep_email':rep_email})
+        d = Context({
+            'group': group,
+            'candidate_name':candidate_name,
+            'group_rep_email':rep_email
+        })
 
-        subject="You're being nominated for endorsement by an official Our Revolution group!"
-        from_email='Our Revolution <info@ourrevolution.com>'
-        to_email=["%s <%s>" % (candidate_name,candidate_email)]
+        subject = "You're being nominated for endorsement by an official Our Revolution group!"
+        from_email = 'Our Revolution <info@ourrevolution.com>'
+        to_email = ["%s <%s>" % (candidate_name, candidate_email)]
+        cc_emails = [
+            "%s <%s>" % (group, rep_email),
+            "%s <%s>" % (
+                'Our Revolution National',
+                settings.ELECTORAL_COORDINATOR_EMAIL
+            )
+        ]
 
         text_content = plaintext.render(d)
         html_content = htmly.render(d)
-        msg = EmailMultiAlternatives(subject, text_content, from_email, to_email)
+        msg = EmailMultiAlternatives(
+            subject,
+            text_content,
+            from_email,
+            to_email,
+            cc=cc_emails
```

Comment:
@cjmabry not sure why this is not working? Seems like should probably support list value? https://github.com/django/django/blob/stable/1.10.x/django/core/mail/message.py